### ISSUE TITLE
Add more interactivity

### ISSUE THREAD

**Issue by andrey-savov** [BartoszJarocki/cv#11]
Some more interactivity would be great.  For example:

1. the skills could be buttons or hover spots that highlight them in the text (by making everything else translucent or slightly blurred).
2. some time line control which can show time as a line with drag support for periods and balloons for projects.
3. professional network as a pop-up graph: people you've directly worked with, interests you follow

Basically the reason to have this instead of a static document is more engagement.  Still needs export to PDF, JSON, etc. but viewers can engage with it.

Of course if you have engagement, you can also measure it. 😂
